index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Paul Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    />
+
+    <link 
+        href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap"
+        rel="stylesheet"
+    />
+</head>
+<body>
+    <header>
+        <nav class="container">
+            <a href="#" class="logo">Paul</a>
+            <i class="material-icons">menu</i>
+        </nav>
+    </header>
+
+    <main>
+      <section class="section">
```

Comment:
As mentioned by the previous reviewer, please make sure to add the background illustration for your section. Because it's required to follow the original Figma design idea.

![image](https://user-images.githubusercontent.com/34779928/198727547-e738b828-1471-48de-b431-29f29dc8c598.png)